available_images.md

```diff
@@ -0,0 +1,140 @@
+
+
+
+    docker pull <name of container image>
+
+ General Framework Containers
+============================
+
+To use the following table, select your desired framework, as well as
+the kind of job you're starting, and the desired Python version. Your
+job type is either ``training`` or ``inference``. Your Python version is
+either ``py27`` or ``py36``. Plug this information into the replaceable
+portions of the URL as shown in the example URL.
+
+You can pin your version by adding the version tag to your URL as follows:
+
+     763104351884.dkr.ecr.us-east-1.amazonaws.com/tensorflow-training:2.2.0-cpu-py37-ubuntu18.04-v1.0
+
+| Framework 			        |  			 Job Type 			            |  			 Horovod Options 			              |  			 CPU/GPU 			 |  			 Python Version Options 			 |  			 Example URL 			                                                                                     | Latest Version |
+|-------------------|------------------------|---------------------------------|------------|---------------------------|----------------------------------------------------------------------------------------------------| --------|
+|  			 TensorFlow 2.2 			 |  			 training 			            | Yes                             |  			 CPU 			     |  			 3.7 (py37) 			             | 763104351884.dkr.ecr.us-east-1.amazonaws.com/tensorflow-training:2.2.0-cpu-py37-ubuntu18.04        | v1.0 |
+|  			 TensorFlow 2.1 			 |  			 inference 			           | No                              |  			 CPU 			     |  			 2.7 (py27), 3.6 (py36) 			 | 763104351884.dkr.ecr.us-east-1.amazonaws.com/tensorflow-inference:2.1.0-cpu-py36-ubuntu18.04       | v6.3 |
+|  			 TensorFlow 2.2 			 |  			 training 			            | Yes                             |  			 GPU 			     |  			 3.7 (py37) 			             | 763104351884.dkr.ecr.us-east-1.amazonaws.com/tensorflow-training:2.2.0-gpu-py37-cu101-ubuntu18.04  | v1.0 |
+|  			 TensorFlow 2.1 			 |  			 inference 			           |  			 No 			                           |  			 GPU 			     |  			 2.7 (py27), 3.6 (py36) 			 | 763104351884.dkr.ecr.us-east-1.amazonaws.com/tensorflow-inference:2.1.0-gpu-py36-cu101-ubuntu18.04 | v6.3 |
+|  			 MXNet 1.6.0 			    |  			 training, inference 			 |  			 Training: Yes, Inference: No 			 |  			 CPU 			     |  			 2.7 (py27), 3.6 (py36) 			 | 763104351884.dkr.ecr.us-east-1.amazonaws.com/mxnet-training:1.6.0-cpu-py36-ubuntu16.04             | v3.5 |
+|  			 MXNet 1.6.0 			    |  			 training, inference 			 |  			 Training: Yes, Inference: No 			 |  			 GPU 			     |  			 2.7 (py27), 3.6 (py36) 			 | 763104351884.dkr.ecr.us-east-1.amazonaws.com/mxnet-training:1.6.0-gpu-py36-cu101-ubuntu16.04       | v3.5 |
+| PyTorch 1.5.0     |  			 training 			            | No                              |  			 CPU 			     |  			 3.6 (py36) 			             | 763104351884.dkr.ecr.us-east-1.amazonaws.com/pytorch-training:1.5.0-cpu-py36-ubuntu16.04           | v1.0 |
```

Comment:
why is inference/training combined for MXNet but not for PyTorch? I understand with TF since the minor versions are different

Reply:
Combined PyTorch entries